.github/workflows/publish.yml

```diff
@@ -10,9 +10,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - uses: ammaraskar/sphinx-action@master
+    - name: Prepare
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y git python3-pip mandoc
+        pip install -r docs/requirements.txt
+    - name: Gen_man_pages
+      run: make man
+    - name: Gen_sphinx
+      uses: ammaraskar/sphinx-action@master
       with:
-        pre-build-command: "apt-get update -y && apt-get install -y git python3-pip && pip install -r docs/requirements.txt"
+        pre-build-command: "apt-get update -y && apt-get install -y git"
```

Comment:
It looks like git is installed earlier already. Likewise below in pull_request.yaml.

Reply:
Unfortunately, `ammaraskar/sphinx-action@master` doesn't use that, looks like it runs in different container with same artifacts. Or maybe I did something wrong.